Tasks/ServiceFabricComposeDeploy/task.json

```diff
@@ -0,0 +1,162 @@
+    "inputs": [
+        {
+            "name": "serviceConnectionName",
+            "type": "connectedService:servicefabric",
+            "label": "Cluster Connection",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Select an Azure Service Fabric service endpoint to be used to connect to the cluster. Choose 'Manage' to register a new endpoint."
+        },
+        {
+            "name": "composeFilePath",
+            "type": "filePath",
+            "label": "Compose File Path",
+            "defaultValue": "**/docker-compose.yml",
+            "required": true,
+            "helpMarkDown": "Path to the compose file that is to be deployed. [Variables](https://go.microsoft.com/fwlink/?LinkID=550988) and wildcards can be used in the path."
+        },
+        {
+            "name": "applicationName",
+            "type": "string",
+            "label": "Application Name",
+            "defaultValue": "fabric:/Application1",
+            "required": true,
+            "helpMarkDown": "Name of the application being deployed."
+        },
+        {
+            "name": "repositoryCredentials",
+            "type": "pickList",
+            "label": "Repository Credentials",
+            "defaultValue": "Endpoint",
+            "required": true,
+            "options": {
+                "Endpoint": "Endpoint",
+                "UsernamePassword": "Username and Password",
+                "None": "None"
+            },
+            "groupname": "repository",
+            "helpMarkDown": "Choose if/how credentials for the docker registry will be provided."
+        },
+        {
+            "name": "dockerRegistryEndpointName",
+            "type": "connectedService:dockerregistry",
+            "label": "Docker Registry Connection",
+            "required": false,
+            "helpMarkDown": "Select a Docker registry connection. If a certificate matching the Server Certificate Thumbprint in the Cluster Connection is installed, it will be used to encrypt the password; otherwise the password will not be encrypted.",
+            "groupname": "repository",
+            "visibleRule": "repositoryCredentials = Endpoint"
+        },
+        {
+            "name": "repositoryUserName",
+            "type": "string",
+            "label": "Repository User Name",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Username for the Docker registry",
+            "groupname": "repository",
+            "visibleRule": "repositoryCredentials = UsernamePassword"
+        },
+        {
+            "name": "repositoryPassword",
+            "type": "string",
+            "label": "Repository Password",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Password for the Docker registry. If the password is not encrypted, it is recomended that you use a secret variable to store it.",
```

Comment:
One spelling error and added some clarification.
"Password for the Docker registry. If the password is not encrypted, it is recommended that you use a custom release definition secret variable to store it.",